libs/util/FormatUtils.arucas

```diff
@@ -0,0 +1,102 @@
+fun repeatStr(string, count){
+    retVal = '';
+    while (count > 0){
+        retVal = retVal+ string;
+        count--;
+    }
+    return retVal;
+}
+class FormatUtil{
+    static fun showMap(map){
+        if (FormatUtil.checkArgs(map)){
+            return "" + map;
```

Comment:
You just don't format it if it contains itself?
Shouldn't you just use a set to check whether a map is within the current evaluation?

Reply:
you mean recursive map such as a = {}; a.put('a', a) ?
pretty cursed but I'll solve it